gossip-bin/src/ui/widgets/mod.rs

```diff
@@ -166,6 +167,34 @@ pub(super) fn set_important_button_visuals(ui: &mut Ui, app: &GossipUi) {
     visuals.widgets.inactive.fg_stroke.color = app.theme.get_style().visuals.extreme_bg_color;
 }
 
+#[allow(dead_code)]
+pub(crate) fn warning_frame<R>(
+    ui: &mut Ui,
+    app: &mut GossipUi,
+    inner: impl FnOnce(&mut Ui, &mut GossipUi) -> R,
+) -> R {
+    egui::Frame::none()
+        .outer_margin(egui::Margin {
```

Comment:
`outer_margin` and `inner_margin` here can be used to change only the warning box